README.md

```diff
@@ -0,0 +1,2 @@
+https://drive.google.com/file/d/1OogCU_qmvH-yhxRAk1pWMAEwSOxFxhkL/view?usp=sharing    
```

Comment:
- **User table**: User can have many cards, but in the same time a card can have only one host, relation is good, but User table configuration has wrong prop: FK CardId. If you have one-to-many relation, FK is needed only on many side (as it is: UserId), just remove the CardId from user table.
-  **Transfer table**: To my mind, to & from CardId looks not bad, but senderCardId and receiverCardId or something like this is better.
- **Transfer - Card relation**: If u have many-to-many relation, u should create the third table to handle, check [here](https://www.datensen.com/blog/er-diagram/many-to-many-relationships/).
- **Deposit - Card relation**: same as above
- **Currency - Card relation**: many-to-many? why? your card have only one currency, should be one-to-one
-  **Exchange table**: Remove FK Currency Id

**ATTENTION**: I was thinking on Currency table, seems like we don't need that table at all, we will have just a enum, feel free to drop that column and remove relation related to the table